SpacesAPI/Space.php

```diff
@@ -302,16 +302,20 @@ public function uploadText(string $text, string $filename, array $params = [], b
      * Upload a file
      *
      * @param string $filepath The path to the file, including the filename. Relative and absolute paths are accepted.
+     * @param string $mimeType The file mime type to pass as ContentType for the file (e.g. 'image/jpeg').
      * @param string|null $filename The remote filename. If `null`, the local filename will be used.
+     * @param bool $private True for the file to be private, false to allow public access.
      *
      * @return \SpacesAPI\File
      */
-    public function uploadFile(string $filepath, ?string $filename = null): File
+    public function uploadFile(string $filepath, string $mimeType, ?string $filename = null, bool $private = true): File
```

Comment:
Please make `$mimeType` optional and change the order of parameters to have it after `$filename`, otherwise this is a breaking change with a major version release, which is not appropriate for a change this size.